cpp/src/parquet/properties.h

```diff
@@ -941,6 +942,15 @@ class PARQUET_EXPORT ArrowReaderProperties {
     return coerce_int96_timestamp_unit_;
   }
 
+  /// Enable Parquet supported Arrow Extension Types.
+  ///
+  /// When enabled, Parquet will use supported Arrow ExtensionTypes in mapping to Arrow
+  /// schema. Currently only arrow::extension::json() extension type is supported. This
+  /// will be used for binary columns whose LogicalType is JSON.
+  void enable_known_arrow_extensions() { known_arrow_extensions_enabled_ = true; }
+  void disable_known_arrow_extensions() { known_arrow_extensions_enabled_ = false; }
+  bool known_arrow_extensions_enabled() const { return known_arrow_extensions_enabled_; }
```

Comment:
Other boolean setters use a single `void set_XXX(bool xxx)` method. Perhaps we should do that here? What do you think @wgtmac ?

Reply:
Renamed.